src/components/ManageAssets/ManageAssets.tsx

```diff
@@ -0,0 +1,161 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import styled from "@emotion/styled";
+import { mixin, vars } from "../../styles";
+import { Dropdown } from "react-bootstrap";
+import { loadAdminAddress } from "../../reducers/admin";
+import { connectToMetamask } from "../../services/etherjs";
+import { AssetInfo } from "../AssetInfo";
+import { FeatureFlag } from "../FeatureFlag";
+
+interface ManageAssetsDropdownProps {
+  userWalletId: string;
+  approvedBeneficiaryAddress: string;
+  approvedHolderAddress: string;
+  beneficiaryAddress: string;
+  holderAddress: string;
+  metamaskAccountError: string;
+  className?: string;
+}
+
+export const ManageAssetsDropdown = styled(
```

Comment:
can take out in a separate file. small lines of code in single file.

Reply:
okay